server/models/listItemModel.js

```diff
@@ -35,12 +35,18 @@ module.exports = {
 
     // After creating the new list item, the job title and company name are added to the list item response object
     listItemModel.create(req.body, function (listItem) {
-      Application.getByAppId(listItem.application_id)
+      console.log('inside add list item, passed in item: ', listItem);
```

Comment:
We'll want to remove the console.logs before we merge, but we can leave them in for now for testing after implementing changes I suggested above.